site/views/overview/index.njk

```diff
@@ -0,0 +1,20 @@
+---
+layout: landing
+name: ESL Presentations
+tags: [draft]
+---
+
+<div class="d-flex container" style="height: 100dvh; ">
+  <div class="m-auto markdown-container">
+    <h1> ESL Presentations </h1>
+    <ul>
+      <li>
+        <a class="btn large btn-link" href="{{ '/overview/quick' | url }}"> Quick Overview </a>
+      </li>
+{#      <li>#}
```

Comment:
Sorry, I missed the difference between quick and full overview

Reply:
There is no full presentation for now (it's going to be detailed dev presentation)
The quick one is the one that we already had used during solutions presentation